asdf/_tests/_helpers.py

```diff
@@ -408,62 +402,3 @@ def assert_no_warnings(warning_class=None):
         assert not any(isinstance(w.message, warning_class) for w in recorded_warnings), display_warnings(
             recorded_warnings,
         )
-
-
-def _assert_extension_type_correctness(extension, extension_type, resolver):
```

Comment:
This was unused so it was removed instead of updating it to not use `get_version_map`.